docs/self-managed/setup/deploy/local/docker-compose.md

```diff
@@ -1,26 +1,212 @@
-Follow the instructions in the [README](https://github.com/camunda/camunda-platform#using-docker-compose).
+A Docker Compose configuration to run Camunda Self-Managed components (e.g., Zeebe, Operate, Tasklist, Optimize, Identity, and Connectors Bundle, etc.).
 
-:::warning
+:::danger
 While the [Docker images](/self-managed/setup/deploy/other/docker.md) themselves are supported for production usage, the Docker Compose files are designed to be used by developers to run an environment locally; they are not designed to be used in production. We recommend to use [Kubernetes](/self-managed/setup/install.md) in production.
 :::
 
-This Docker Compose configuration serves two purposes:
+## Prerequisites
+
+- **Docker Compose:** Version 1.27.0+ (supports the [latest compose specification](https://docs.docker.com/compose/compose-file/)).
+- **Docker:** Version 20.10.16+.
+- **Local:** Add Keycloak to resolve to 127.0.0.1 on your local machine, and set `KEYCLOAK_HOST=keycloak` in the `.env` file for token refresh and logout functionality.
+
+## Setup
+
+### Start
+
+To spin up a Camunda Platform 8 Self-Managed environment locally you can use the following docker compose configuration files:
+
+- **docker-compose.yaml:** Contians all Camunda 8 Components for a full stack deployment: Zeebe, Operate, Tasklist, Connectors, Optimize, Identity, Elasticsearch, Keycloak, Web Modeler, and PostgreSQL.
+- **docker-compose-core.yaml:** Contians Camunda 8 Orchestration cluster components: Zeebe, Tasklist, Operate, Optimize, Identity and Connectors.
+- **docker-compose-web-modeler.yaml:** Contians Camunda 8 Web Modeler standalone installation.
+
+To start a complete Camunda Platform 8 Self-Managed environment locally:
+
+1. Clone this repository.
+2. Run:
+   ```shell
+   docker compose up -d
+   ```
+3. Wait for the environment to initialize. Monitor logs (especially Keycloak) to ensure all components have started.
+
+Wait a few minutes for the environment to start up and settle down. Monitor the logs, especially the Keycloak container log, to ensure the components have started.
+
+Now you can navigate to the different web apps and log in with the user `demo` and password `demo`:
+
+- Operate: [http://localhost:8081](http://localhost:8081)
+- Tasklist: [http://localhost:8082](http://localhost:8082)
+- Optimize: [http://localhost:8083](http://localhost:8083)
+- Identity: [http://localhost:8084](http://localhost:8084)
+- Elasticsearch: [http://localhost:9200](http://localhost:9200)
+
+Keycloak is used to manage users. Here you can log in with the user `admin` and password `admin`:
+
+- Keycloak: [http://localhost:18080/auth/](http://localhost:18080/auth/)
+
+The workflow engine Zeebe is available using gRPC at `localhost:26500`.
+
+### Stop
+
+```shell
+docker compose down -v
```

Comment:
I think we should have a disclaimer about the -v flag destroying persistent data.

Reply:
Actually this looks covered already further down